platforms/common/include/px4_platform_common/i2c_spi_buses.h

```diff
@@ -102,13 +138,12 @@ class BusCLIArguments
 
 
 	I2CSPIBusOption bus_option{I2CSPIBusOption::All};
-	uint16_t type{0}; ///< device type (driver-specific)
 	int requested_bus{-1};
 	int chipselect_index{1};
 	Rotation rotation{ROTATION_NONE};
 	int bus_frequency{0};
 	spi_mode_e spi_mode{SPIDEV_MODE3};
-	uint8_t i2c_address{0}; ///< optional I2C address: a driver can set this to allow configuring the I2C address
+	uint8_t i2c_address{0}; ///< I2C address (a driver must set the default address)
```

Comment:
Not necessarily here, but later could you clarify how we should be handling drivers that support a range of I2C addresses?

Reply:
I see 2 main patterns:
- start multiple independent instances with different addresses
- let the driver handle it internally and cycle through the addresses (so it's not configurable), the mb12xx is an example. If we need more configurability there, we can extend the base, but at this point I don't think it's needed.